src/components/BuildConfigsList/BuildConfigsList.tsx

```diff
@@ -27,13 +27,15 @@ import { Username } from 'components/Username/Username';
 
 import { areDatesEqual, checkColumnsCombinations, createDateTime } from 'utils/utils';
 
+type TColumns = Array<keyof typeof buildConfigEntityAttributes>;
+
 interface IBuildConfigsListProps {
   serviceContainerBuildConfigs: IServiceContainer;
-  columns?: Array<keyof typeof buildConfigEntityAttributes>;
+  columns?: TColumns;
```

Comment:
Shall we also change it for the artifacts list to make it universal? https://github.com/project-ncl/pnc-web-ui-react/blob/main/src/components/ArtifactsList/ArtifactsList.tsx#L44

Reply:
Yes, I know about this one, can you create separated jira for that?